scripts/valhalla_build_timezones

```diff
@@ -1,18 +1,16 @@
 #!/bin/sh
 
 error_exit() {
-  echo "$1" 1>&2
-  #exit 1
+  echo "error: $1" 1>&2
```

Comment:
Add "error" to emphasize that this is a critical error (and to highlight it where applicable)

Reply:
The reason this is commented is because of recent versions of libgeos double freeing some memory. In others words, on some platforms this breaks the build. The double free happens at the end of the operation so it's harmless but erroring caused everything else to fail